src/index.html

```diff
@@ -1,17 +1,152 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="http://localhost:3000/api" method="post">
+      <fieldset class="personal">
+        <legend>Personal information</legend>
+        <label for="surname">Surname:</label>
+        <input
+          type="text"
+          id="surname"
+          name="surname"
+          autocomplete="off"
+        >
+        <br>
+        <label for="name">Name:</label>
+        <input
+          type="text"
+          id="name"
+          name="name"
+          autocomplete="off"
+        >
+        <br>
+        <label for="how_old">How old are You?</label>
+        <input
+          type="number"
+          id="how_old"
+          name="how_old"
+          min="1"
+          max="100"
+          value="12"
+        >
+        <br>
+        <label for="birth">Full date of birth:</label>
+        <input
+          type="date"
+          id="birth"
+          name="birth"
+        >
+        <br>
+        <label for="term">I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="term"
+            id="term"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="registration">
+        <legend>Registration</legend>
+        <label for="email">E-mail:</label>
+        <input
+          type="email"
+          id="email"
+          name="email"
+          placeholder="email@example.com"
+          autocomplete="off"
+          required
+        >
+        <br>
+        <label for="passwoprd">Password:</label>
+        <input
+          type="passwoprd"
+          id="passwoprd"
+          name="passwoprd"
+          minlength="5"
+          maxlength="10"
+          autocomplete="off"
+        >
+      </fieldset>
+      <fieldset class="fact">
+        <legend>An interesting fact about you!</legend>
+        <div class="add-cats-margin">
+          <span>Do you love cats?</span>
+          <input
+          id="love_cats_y"
+          type="radio"
+          name="love_cats"
+          value="y"
+          >
+          <label for="love_cats_y">Yes</label>
+          <input
```

Comment:
you can move all inputs to labels

Reply:
Ok, next time, plz.
I see it is one of the option: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label